server/queue.go

```diff
@@ -605,11 +606,33 @@ func (s *Server) clientMadeIRODSConnections(c *gin.Context) {
 	defer s.mapMu.Unlock()
 
 	s.iRODSConnections[hostPID] += n
-	s.sendSlackMessage(slack.Info, strconv.Itoa(s.totalIRODSConnections())+" iRODS connections open")
+	s.createAndSendIRODSSlackMsg()
 
 	c.Status(http.StatusOK)
 }
 
+func (s *Server) createAndSendIRODSSlackMsg() {
+	msg := fmt.Sprintf("%d iRODS connections open", s.totalIRODSConnections())
+
+	if s.iRodsTracker.slacker == nil || s.iRodsTracker.bouncing || msg == s.iRodsTracker.lastMsg {
+		return
+	}
+
+	s.iRodsTracker.slacker.SendMessage(slack.Info, msg)
+	s.iRodsTracker.lastMsg = msg
+	s.iRodsTracker.bouncing = true
+	debounce := s.iRodsTracker.debounce
+
+	go func() {
+		<-time.After(debounce)
```

Comment:
I don't think that there's much difference performance wise, but using `time.Sleep(debounce)` is probably more idiomati, and will be slightly more efficient.